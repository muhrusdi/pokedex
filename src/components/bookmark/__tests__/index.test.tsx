import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Button from "../";

describe("Button Bookmark Component", () => {
  let toggle = false;

  const handleClick = () => {
    toggle = !toggle;
  };
  const { getByRole, rerender } = render(
    <Button toggle={toggle} onClick={handleClick} />
  );
  const button = getByRole("button");
  test("Should be white color by default", () => {
    // Assertion
    expect(button.querySelector("svg").getAttribute("fill")).toEqual("#fff");
  });
});
