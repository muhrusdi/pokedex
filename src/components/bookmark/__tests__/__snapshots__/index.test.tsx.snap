// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays the correct title 1`] = `
<h1
  data-testid="hero-title"
>
  Gatsby is awesome!
</h1>
`;
